### ISSUE TITLE
Square-root in EuclideanDistanceCalculator.java

### ISSUE THREAD

**Issue by steff1193** [amaurycrickx/recognito#14]
In distances/EuclideanDistanceCalculator.java method `getDistance`, you sum the squares of all the diffs. Shouldnt you do square-root on this sum before returning?

**Comment by amaurycrickx** [comment]
you're right, I guess this shouldn't be called Euclidean distance anymore

**Comment by amaurycrickx** [comment]
Do you think I'm right in believing that not applying square-root on the sum helps keeping spread bigger and thus helps when more voice prints are added to the system?

**Comment by steff1193** [comment]
Yeah, I had the same thought, that, they way it is used, it probably makes things better to leave it as it is. But you should use a different name, because some day, when used in another context, it may be important that it is actually correct euclidean distance that is calculated. If nothing else, it is just really bad naming :-)
